app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,59 @@
+class CompaniesController < ApplicationController
+  before_action :company, only: %i[show edit update destroy]
+
+  def index
+    @search = Company.search(params[:q])
```

Comment:
![image](https://user-images.githubusercontent.com/33605808/95783858-63ff9f00-0cdb-11eb-90b4-8dc731aa41fa.png)

Reply:
one more, please add pagination to companies, it is bad if you will load all companies on the page